### ISSUE TITLE
Multiple packages setup

### ISSUE THREAD

**Issue by clomie** [clomie/gas-vue-typescript#18]
Would be better to have multiple package setup like yarn workspaces.
https://classic.yarnpkg.com/en/docs/workspaces

Want to do:
- Run npm scripts on both packages (frontend and script) from root directory in a platform-independent way.
- Enable linting and type checking on edit time with VSCode.

**Comment by clomie** [comment]
Tried yarn workspaces but type checking failed on frontend package due to `@types/google-apps-script` package.
Same error as described here: https://github.com/DefinitelyTyped/DefinitelyTyped/issues/32585

This is because the yarn workspaces installs npm modules of all package into the one node_modules directory. The `@types/google-apps-script` module is installed that is not required for frontend package but required for script package. This causes conflict in the variable `console` when the type checking on frontend package.

To avoid this error, need keeping node_modules directory are separated.
So I decided to not setup yarn workspaces.
